### DISCUSSION TITLE
How to transfer sale-price, sale-start-date, sale-end-date

### DISCUSSION THREAD

**Discussion by Lemodo20** [amzn/selling-partner-api-models#3659] [General]
Hi,

in old MWS Flatfiles i can transfer above fields to amazon. But new specification don't allow them any more.

So my old file looks like this:
TemplateType=Offer	Version=2014.0703
sku	price	quantity	operation-type	sale-price	sale-start-date	sale-end-date	leadtime-to-ship	fulfillment-center-id

new Flat.File.PriceInventory looks like this:
sku	price	minimum-seller-allowed-price	maximum-seller-allowed-price	quantity	fulfillment-channel	handling-time

so this 3 fields are removed. 

How can i transfer now this 3 informations. I don't find other flatfiles, which contain this fields.

**Comment by BestTrolley** [comment]
The standard item feed template still contains those. (using partial_update)
